index.bs

```diff
@@ -2702,11 +2702,14 @@ Determining the W3C Decision</h4>
 	those other conditions must also be re-fulfilled.
 
 	<div class="example">
-		For example, to make [=substantive changes=] to a [=Proposed Recommendations=],
-		the [=technical report=] could be returned to [=Candidate Recommendation=].
+		For example, to make [=substantive changes=] to a [=technical report=]
+		prior to publication as a [=Recommendation=],
+		it would need to go through a new [=Update Request=],
+		be republished as a new [=Candidate Recommendation=],
+		and fulfill the criteria for advancement.
 		Alternatively, the desired changes can be introduced as non-substantive amendments
 		using the process for [[#revising-rec|revising a Recommendation]].
-		However, they cannot be directly integrated between [=PR=] and [=REC=],
+		However, they cannot be directly integrated between [=CRS=] and [=REC=],
```

Comment:
Except for removal of at-risk features.

Reply:
True. However, is that too much detail for an example? We could add it as a parenthetical, but I am not sure if it really brings clarity.

Reply:
I don't think it's too much detail. Without mentioning this, it looks like the example contradicts the Process, and that could be confusing for some group later if they do want to remove at-risk features at this stage.